apps/ios/LiveVideo/LiveVideo/Views/Stream/SpeakerGridView.swift

```diff
@@ -45,8 +46,24 @@ struct SpeakerGridView: View {
                 GeometryReader { geometry in
                     LazyVGrid(columns: columns, spacing: spacing) {
                         ForEach($viewModel.onscreenSpeakers, id: \.self) { $speaker in
-                            SpeakerVideoView(speaker: $speaker)
-                                .frame(height: geometry.size.height / CGFloat(rowCount) - spacing)
+                            Group {
+                                if role == .host && !speaker.isMuted && !speaker.isYou  {
+                                    Menu(
```

Comment:
Does wrapping the `SpeakerVideoView` in `Menu` enable long clicking the video to mute?

Reply:
A single tap will bring up the "menu".

I could actually use a "context menu" instead which would be a long press and then it blurs everything on the screen except that view and the menu. But I think this is typically used when a single tap is already used for another operation like present a new screen. Or if it is an advanced feature that we want to make harder to get to.

The regular menu seemed like the way to go.